src/tsolvers/lrasolver/LRASolver.C

```diff
@@ -191,6 +191,32 @@ LRASolver::~LRASolver( )
 
 LRALogic&  LRASolver::getLogic() { return logic; }
 
+lbool LRASolver::getPolaritySuggestion(PTRef ptref) const {
+    if (!this->isInformed(ptref)) { return l_Undef; }
+    LVRef var = this->lavarStore.getVarByLeqId(logic.getPterm(ptref).getId());
```

Comment:
Do we need a method LVRef getVarByPTRef(PTRef tr) const { return getVarByLeqId(logic.getPterm(tr).getId())); }  ?

Reply:
> Do we need a method LVRef getVarByPTRef(PTRef tr) const { return getVarByLeqId(logic.getPterm(tr).getId())); } ?

It looks like it could be useful. I can try to add it.